### ISSUE TITLE
Hiring-Page 

### ISSUE THREAD

**Issue by gunjan-g** [2024-SANDHYA/Dust-Road-Co#43]
### Description
Creating a we are hiring page for website.

**Comment by Pushti-R** [comment]
Hey @gunjan-g  can I work on this issue ?

**Comment by gunjan-g** [comment]
> Hey @gunjan-g can I work on this issue ?

Actually this has been suggested by someone else, so he would be assigned first

**Comment by Pushti-R** [comment]
Can you brief a little about this task ?
or some design which you were thinking

**Comment by gunjan-g** [comment]
> Can you brief a little about this task ? or some design which you were thinking

This issue has been suggested by one of the contributors, so he'll be knowing about his idea and design

**Comment by Pushti-R** [comment]
Okay thank you

**Comment by AmanRana10** [comment]
I would like to work on this issue .... pls assign this to me

**Comment by AmanRana10** [comment]
#49 added hiring page

**Comment by gunjan-g** [comment]
> #49 added hiring page

please send pr link on discord

**Comment by radzhiv25** [comment]
is the issue still available or what

**Comment by gunjan-g** [comment]
> is the issue still available or what

Yes, it's available. I would assign you if you want to work on it

**Comment by radzhiv25** [comment]
> > is the issue still available or what
> 
> Yes, it's available. I would assign you if you want to work on it

is it to be integrated in the same page or separate page is to be created for the same

**Comment by gunjan-g** [comment]
> > > is the issue still available or what
> > 
> > 
> > Yes, it's available. I would assign you if you want to work on it
> 
> is it to be integrated in the same page or separate page is to be created for the same

you can create separate page and then add it in navigation bar

**Comment by radzhiv25** [comment]
ok then you can assign it to me 
also please mention the deadline too

**Comment by gunjan-g** [comment]
> ok then you can assign it to me also please mention the deadline too

You can do it as soon as possible, but we will unassign you if you didn't complete it in 10-12 days

**Comment by gunjan-g** [comment]
@2024-SANDHYA Please assign @radzhiv25

**Comment by 2024-SANDHYA** [comment]
@radzhiv25 You already have an issue opened & PR too. Work on that first.
